astropy/time/tests/test_methods.py

```diff
@@ -620,6 +621,35 @@ def test_sort(self, use_mask):
             assert np.all(self.t0.sort(-1)[:, :, 0] == self.t0.min(-1))
             assert np.all(self.t0.sort(-1)[:, :, -1] == self.t0.max(-1))
 
+    def test_mean(self, use_mask):
```

Comment:
Could you also add a test that exercises the various `axis` options?  `self.t0` does not have a location, `self.t1` a scalar one, and `self.t2` an array-valued one, so that is handy here...

Plus perhaps a test of `where`...